tests/api/test_endpoints.py

```diff
@@ -95,6 +95,31 @@ def test_put_users_update_password__success(server):
     assert response.status == 200
 
 
+def test_put_users_update_password__failure(server):
+    username = 'test'
+    token = util.create_jwt(1, server.config.secret)
+    _, response = server.app.test_client.put(
+        f'/users/{username}',
+        data=json.dumps({
+            'password': 'Val1dPassword!123',
+        }),
+        headers={'Authorization': token})
+    assert response.status == 400
+
+
+def test_put_users_update_password__success(server):
+    username = 'test'
+    token = util.create_jwt(1, server.config.secret)
+    _, response = server.app.test_client.put(
+        f'/users/{username}',
+        data=json.dumps({
+            'old_password': 'Val1dPassword!',
+            'password': 'Val1dPassword!s',
+        }),
+        headers={'Authorization': token})
+    assert response.status == 200
```

Comment:
Did you mean to include this? It looks like these two functions are duplicates of the functions below.

Reply:
Oh I might have rebased this improperly.  Sorry about that.